components/cam/cime_config/config_component.xml

```diff
@@ -27,15 +27,15 @@
   </entry>
 
   <entry id="CAM_TARGET">
-         <type>char</type>
-         <valid_values>preqx,preqx_kokkos,preqx_acc,theta-l</valid_values>
-         <default_value>preqx</default_value>
-	 <values>
+    <type>char</type>
+    <valid_values>preqx,preqx_kokkos,preqx_acc,theta-l</valid_values>
+    <default_value>preqx</default_value>
```

Comment:
Do we really want preqx to be the default here?

Reply:
I think we inherit that from E3SM. We can change it.